src/Filters/Block.php

```diff
@@ -420,6 +420,14 @@ private function getQuantitiesBlock($filter, $selectedFilters)
                 'nbr' => 0,
             ],
             1 => [
+                'name' => $this->context->getTranslator()->trans(
```

Comment:
This option should be available only when the stock management is activated, no?

Reply:
If you disable stock management, merchant cannot update quantity nor set out-of-stock. So the quantity block is not needed at all.

It worked before, but I modified it a bit. Maybe it will be some miliseconds faster this way.